DMX.Core.Api.Tests.Unit/Services/Orchestrations/LabOrchestrationServiceTests.Exceptions.Remove.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DMX.Core.Api.Models.Foundations.Labs;
+using DMX.Core.Api.Models.Orchestrations.Labs.Exceptions;
+using FluentAssertions;
+using Moq;
+using Xeptions;
+using Xunit;
+
+namespace DMX.Core.Api.Tests.Unit.Services.Orchestrations
+{
+    public partial class LabOrchestrationServiceTests
+    {
+        [Theory]
+        [MemberData(nameof(LabDependencyValidationExceptions))]
+        public async Task ShouldThrowOrchestrationDependencyValidationExceptionOnRemoveIfDependencyValidationErrorOccursAndLogItAsync(
+            Xeption validationException)
+        {
+            // given
+            Guid randomGuid = Guid.NewGuid();
```

Comment:
Since this is just a placeholder and we are going to throw the exception anyway. call this `someLabId`

Reply:
Do the same for the rest.